### PR TITLE
docs(readme): correct examples link

### PR DESCRIPTION
**Pull request by HoKim98**
Hello, this PR just corrects the examples hyperlink.
The PR #1459 introduced the workspace directory, but it missed this link.

* Before: https://github.com/ratatui/ratatui/tree/main/examples/README.md
* After: https://github.com/ratatui/ratatui/tree/main/ratatui/examples/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -363,7 +363,7 @@ for a curated list of awesome apps/libraries built with `ratatui`!
 
 ## Alternatives
 
-You might want to checkout [Cursive](https://github.com/gyscos/Cursive) or
```
Comment (orhun) [inline README.md:366]: Can you revert this formatting change? It seems unrelated :)

Comment (joshka) [reply README.md:366]: I'm fine with this - the EOL space doesn't belong.

**Review by joshka — CHANGES_REQUESTED** [review]
It may not be obvious, but we generate most of the readme from the lib.rs using cargo-rdme. Can you please also make the same changes there also so that these keep in sync with each other. https://github.com/ratatui/ratatui/actions/runs/11801844681/job/32876120141?pr=1484 is the failing CI check for this.

**Comment by HoKim98** [comment]
Oh it's my mistake

File: README.md (hunk 1 above)
Comment (HoKim98) [reply README.md]: I think this PR is not relevant to this part. It's my fault by using auto-formatter.

**Review by joshka — APPROVED** [review]
